Fuse color sections for named-color (#24450)
mdn/content: files/en-us/web/css/named-color/index.md

### Value

Named colors consists of standard colors, and of the `transparent` keyword.

#### Standard colors

> The following colors have a keyword associated to them:

Reviewer: ```suggestion
The following colors have a keyword associated with them:
```

### Value

> Named colors consists of standard colors, and of the `transparent` keyword.

Author: ```suggestion
Named colors consists of standard colors, the `transparent` and [`currentcolor`](/en-US/docs/Web/CSS/color_value#currentcolor_keyword) keywords.
```

      <td style="background: forestgreen"></td>
    </tr>
    <tr>
      <td><code>fuchsia</code></td>
      <td><code>#ff00ff</code></td>
      <td style="background: fuchsia"></td>
    </tr>
    <tr>
      <td><code>gainsboro</code></td>
      <td><code>#dcdcdc</code></td>
      <td style="background: gainsboro"></td>
    </tr>
    <tr>
      <td><code>ghostwhite</code></td>
      <td><code>#f8f8ff</code></td>
      <td style="background: ghostwhite"></td>
    </tr>
    <tr>
      <td><code>gold</code></td>
      <td><code>#ffd700</code></td>
      <td style="background: gold"></td>
    </tr>
    <tr>
      <td><code>goldenrod</code></td>
      <td><code>#daa520</code></td>
      <td style="background: goldenrod"></td>
    </tr>
    <tr>
      <td><code>gray</code></td>
      <td><code>#808080</code></td>
      <td style="background: gray"></td>
    </tr>
    <tr>
      <td><code>green</code></td>
      <td><code>#008000</code></td>
      <td style="background: green"></td>
    </tr>
    <tr>
      <td><code>greenyellow</code></td>
      <td><code>#adff2f</code></td>
      <td style="background: greenyellow"></td>
    </tr>
    <tr>
      <td><code>grey</code></td>

>       <td><code>#808080</code></td> (synonym of <code>gray</code>)

Reviewer 2: ```suggestion
      <td><code>#808080</code> (synonym of <code>gray</code>)</td>
```

      <td style="background: paleturquoise"></td>
    </tr>
    <tr>
      <td><code>palevioletred</code></td>
      <td><code>#db7093</code></td>
      <td style="background: palevioletred"></td>
    </tr>
    <tr>
      <td><code>papayawhip</code></td>
      <td><code>#ffefd5</code></td>
      <td style="background: papayawhip"></td>
    </tr>
    <tr>
      <td><code>peachpuff</code></td>
      <td><code>#ffdab9</code></td>
      <td style="background: peachpuff"></td>
    </tr>
    <tr>
      <td><code>peru</code></td>
      <td><code>#cd853f</code></td>
      <td style="background: peru"></td>
    </tr>
    <tr>
      <td><code>pink</code></td>
      <td><code>#ffc0cb</code></td>
      <td style="background: pink"></td>
    </tr>
    <tr>
      <td><code>plum</code></td>
      <td><code>#dda0dd</code></td>
      <td style="background: plum"></td>
    </tr>
    <tr>
      <td><code>powderblue</code></td>
      <td><code>#b0e0e6</code></td>
      <td style="background: powderblue"></td>
    </tr>
    <tr>
      <td><code>purple</code></td>
      <td><code>#800080</code></td>
      <td style="background: purple"></td>
    </tr>
    <tr>
      <td>

>         <a href="https://codepen.io/trezy/post/honoring-a-great-man"

Reviewer 2: ```suggestion
        <a href="https://meyerweb.com/eric/thoughts/2014/06/19/rebeccapurple/"
```

So we don't have a duplicate link.

Reviewer 2: let's switch the other one, since the other one mentions him by name so we should link to his blog.

Author: So I'm keeping this post here, and change the link to the blog at the bottom

>         <a href="https://codepen.io/trezy/post/honoring-a-great-man"
>           ><code>rebeccapurple</code></a
>         >
>       </td>

Reviewer 2: ```suggestion
        <code>rebeccapurple</code>
      </td>
```

Reviewer 2: Let's just get rid of this here as it's confusing when only one color has a link.

    </tr>
    <tr>
      <td><code>transparent</code></td>
      <td>See <a href="#transparent">transparent</a>.</td>
      <td style="background: transparent"></td>
    </tr>
    <tr>
      <td><code>turquoise</code></td>
      <td><code>#40e0d0</code></td>
      <td style="background: turquoise"></td>
    </tr>
    <tr>
      <td><code>violet</code></td>
      <td><code>#ee82ee</code></td>
      <td style="background: violet"></td>
    </tr>
    <tr>
      <td><code>wheat</code></td>
      <td><code>#f5deb3</code></td>
      <td style="background: wheat"></td>
    </tr>
     <tr>
      <td><code>white</code></td>
      <td><code>#ffffff</code></td>
      <td style="background: white"></td>
    </tr>
    <tr>
      <td><code>whitesmoke</code></td>
      <td><code>#f5f5f5</code></td>
      <td style="background: whitesmoke"></td>
    </tr>
    <tr>
      <td><code>yellow</code></td>
      <td><code>#ffff00</code></td>
      <td style="background: yellow"></td>
    </tr>
    <tr>
      <td><code>yellowgreen</code></td>
      <td><code>#9acd32</code></td>
      <td style="background: yellowgreen"></td>
    </tr>
    <tr>
  </tbody>
</table>

> Initially, in [CSS Level 1](https://www.w3.org/TR/REC-CSS1/), only 16 basic colors were defined, with `orange` added in [CSS Level 2](https://www.w3.org/TR/CSS2/). Web designers found this list too short, and browser vendors added numerous names for colors based on the X11 color names. In [SVG 1.0](https://www.w3.org/TR/2001/REC-SVG-20010904/), then in [CSS Colors Level 3](https://www.w3.org/TR/css-color-3/), these names got standardized, formally defined, and uniformized (some had different spellings that are now aliases). They are called the _extended color keywords_, the _X11 colors_, or the _SVG colors_.

Reviewer 2: ```suggestion
Initially, in [CSS Level 1](https://www.w3.org/TR/REC-CSS1/), only 16 basic colors were defined, with `orange` added in [CSS Level 2](https://www.w3.org/TR/CSS2/). Web designers found this list too short, and browser vendors added numerous names for colors based on the X11 color names. In [SVG 1.0](https://www.w3.org/TR/2001/REC-SVG-20010904/), then in [CSS Colors Level 3](https://www.w3.org/TR/css-color-3/), these names got standardized, formally defined, and made uniform (some had different spellings that are now aliases). They are called _extended color keywords_, _X11 colors_, or _SVG colors_.
```

> In [CSS Colors Level 4](https://www.w3.org/TR/css-color-4/), an additional color, `rebeccapurple` was added [to honor web pioneer Eric Meyer](https://codepen.io/trezy/post/honoring-a-great-man), who suffered a terrible personal tragedy.

Reviewer 2: ```suggestion
In [CSS Colors Level 4](https://www.w3.org/TR/css-color-4/), an additional color, `rebeccapurple` was added to honor [web pioneer Eric Meyer](https://meyerweb.com/eric/thoughts/2014/06/19/rebeccapurple/), who suffered a terrible personal tragedy.
```

> In [CSS Colors Level 4](https://www.w3.org/TR/css-color-4/), an additional color, `rebeccapurple` was added to honor [web pioneer Eric Meyer](https://meyerweb.com/eric/thoughts/2014/06/19/rebeccapurple/), who suffered a terrible personal tragedy.

Reviewer 2: ```suggestion
In [CSS Colors Level 4](https://www.w3.org/TR/css-color-4/#named-colors), an additional color, `rebeccapurple` was added to honor [web pioneer Eric Meyer](https://meyerweb.com/eric/thoughts/2014/06/19/rebeccapurple/), who suffered a terrible personal tragedy.
```

> ### transparent

Reviewer 2: should this be a capital letter?

Author: No this is the keyword
